core/server/services/auth/session.js

```diff
@@ -0,0 +1,75 @@
+const config = require('../../config'),
+    {User} = require('../../models/user'),
+    db = require('../../data/db'),
+    session = require('express-session'),
+    KnexSessionStore = require('connect-session-knex')(session),
+    {BadRequestError, UnauthorizedError, InternalServerError} = require('../../lib/common/errors');
+
+const createSession = function createSession(req, res, next) {
+    if (!req.body) {
+        return next(new BadRequestError());
+    }
+    const {username, password} = req.body;
+    return User.check({
+        email: username,
+        password
+    }).then(function (user) {
+        req.session.user_id = user.id;
+        req.session.save(function (err) {
+            if (err) {
+                return res.send(500);
+            }
+            res.send(201, {});
```

Comment:
This was a funny lil thing where ember was expecting JSON and going nuts if it didn't get any.